lib/src/sdk_env.dart

```diff
@@ -297,15 +297,17 @@ class ToolEnvironment {
   Future<Map<String, dynamic>> getFlutterVersion() async {
     final result = _handleProcessErrors(
         await runProc([..._flutterCmd, '--version', '--machine']));
-    var content = result.asJoinedOutput;
     final waitingForString = 'Waiting for another flutter';
```

Comment:
Can't we avoid this if we use `FLUTTER_ROOT=/path/to/flutter dart pub outdated` instead?

Reply:
I think this happened sporadically in CI, and while we've changed the flutter setup since, we've always had `FLUTTER_ROOT` set.

Reply:
My point is: I don't think `dart pub outdated` will ever print this line. But it's possible that `flutter pub outdated` will, so there is no need for this special case if we use `dart pub outdated` with `FLUTTER_ROOT` instead.

Reply:
Yeah, we've always run `dart pub outdated` (there was a PR to change that, but got reverted), however, this is `flutter --version --machine`, which did write this on the output, on CI. Let's keep it for a bit, and maybe do a more systemic review of the code later: we should be able to identify similar patterns in multiple places, which may have become obsolete over time.